tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES6",
```

Comment:
:eyes:

Reply:
Main reason for this change?

Reply:
I suggested it a while ago. I'll pose the opposite question, why would we target ES6? We're using modern node runtimes. We don't need to worry about IE11 compatibility or anything

Reply:
Yeah we should be targetting the newest build + it's required for the part of our code that computes with BigInts as those apparently don't like ES6